server/test_helpers/fake-data.js

```diff
@@ -108,7 +111,7 @@ export const FAKE_RECORD_ONLY_LOW_TEST = MarcRecord.fromString([
 export const FAKE_CYRILLIC_RECORD = MarcRecord.fromString(`LDR    01731cam a22002417i 4500
 001    002686913
 005    20050825143756.0
-008    050712s2005    ru      r     000 u rus  
+008    050712s2005    ru      r     000 u rus
```

Comment:
Field 008 should end in 3 spaces, does removing the spaces break the test record?